test/test_components.jl

```diff
@@ -1,9 +1,15 @@
 @testset "Shapes" begin
+    Shape = Plots.Shape
+    brush = Plots.brush
+    get_xs = Plots.Shapes.get_xs
+    get_ys = Plots.Shapes.get_ys
+    vertices = Plots.Shapes.vertices
+    rotate! = Plots.Shapes.rotate!
```

Comment:
apart from Shape we should still export these

Reply:
do you mean that Plots should be exporting these to public, or Shapes exporting these to Plots?

Reply:
> do you mean that Plots should be exporting these to public

That, its documented API und should stay that way IMO

Reply:
true, ill update the export, can't find where was this orginally exported